portfolio/src/main/java/com/google/sps/servlets/fetchServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
change the javadoc to /fetch here

Reply:
Also since your random function is to generate a song, you can just use /secret-song as the URL maybe. /fetch is from the walkthrough but it's a bit more general